js/global-eval.ts

```diff
@@ -1,6 +1,10 @@
-// If you use the eval function indirectly, by invoking it via a reference
-// other than eval, as of ECMAScript 5 it works in the global scope rather than
-// the local scope. This means, for instance, that function declarations create
-// global functions, and that the code being evaluated doesn't have access to
-// local variables within the scope where it's being called.
+/**
```

Comment:
Hmmm... this shouldn't be part of this PR... seems like something strange with the rebasing again.